metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'cookbooks@opscode.com'
 license           'Apache 2.0'
 description       'Installs and configures nginx'
-version           '2.7.4'
+version           '2.7.5'
```

Comment:
do not increment version numbers, this is done on release by the maintainers only.